spec/features/data_access/single_granule_access_spec.rb

```diff
@@ -36,10 +36,21 @@
       expect(page).to have_content "1 Granule"
     end
 
+    it 'does not have granule filter in back url' do
+      expect(find_link('Back to Search Session')[:href]).not_to include("&sgd=")
+    end
+
     it 'limits the data access to only the selected granule' do
       click_link 'Expand List'
+      wait_for_xhr
       expect(page).to have_content 'FIFE_RAIN_30M.72981621.r30'
     end
+
+    it 'does not filter the granules upon returning to the collection' do
```

Comment:
We try not to follow links within an `it` statement. You would want a new context `context 'when returning to the search results' do` to handle this. Inside the context will be a `before` block that does the `click_link` and then and `it` that asserts the list is correct